src/js/form.js

```diff
@@ -828,7 +827,7 @@ Form.prototype.pathToAbsolute = function( targetPath, contextPath ) {
 
 /**
  * Validates question values.
- * 
+ *
  * @param  {jQuery} $input    [description]
  * @return {Promise}           [description]
```

Comment:
This is just a question, but I'm wondering if JSDoc has a way to indicate what the data type of the final resolved value is.